crates/bevy_ecs/Cargo.toml

```diff
@@ -24,6 +24,7 @@ async-channel = "1.4"
 event-listener = "2.5"
 thread_local = "1.1.4"
 fixedbitset = "0.4.2"
+fxhash = "0.2"
```

Comment:
As mentioned before, `fxhash` is no longer really maintained. Could you change this out for `rustc-hash`?

Reply:
Done